GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -1028,13 +1034,14 @@ protected void invokeUrl(String serverBase, String urlString,
 
             Row headerRow = sheet.getRow(headerRowIndex);
             boolean firstQuestionFound = false;
-            int firstQuestionColumnIndex = 0;
+            int firstNonheaderColumnIndex = 0;
+            int lastNonemptyHeaderColumnIndex = 0;
```

Comment:
I propose we initialize it to `-1` to indicate that no column has been found rather than defaulting to the first column. 

```suggestion
            int lastNonemptyHeaderColumnIndex = -1;
```

Reply:
So do you then want to get rid of the firstQuestionFound variable? Because that is doing that job right now.

Reply:
Good observation and yes. it would not be needed anymore actually! 👍